ONSdigital/eq-survey-runner
# Remove previous accessibility tests

**_What**_

Removes previous accessibilty tests from repo
Adds a11y test dir to repo, ignoring contents

**_How to test**_

1) Checkout
2) run `npm install`
3) Run accessibility tests with `gulp test:a11ym`
4) Run git status and check there are no changes in `test/a11y/`

**_Who can test**_

Anybody, but perhaps @hamishtaplin should take a look
